fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexer.java

```diff
@@ -1094,7 +1110,7 @@ public static class Builder {
         private IndexingPolicy indexingPolicy = null;
         private IndexingPolicy.Builder indexingPolicyBuilder = null;
         @Nullable
-        private Function<Config, Config> configLoader = null;
+        private UnaryOperator<Config> configLoader = null;
```

Comment:
This is a superficial change, unless someone is passing in a variable of type `Function<Config, Config>`, correct? I would expect most uses to be a lambda, or method reference, and it's `UNSTABLE`, so I think this makes sense.

Reply:
I believe so. 
https://stackoverflow.com/questions/54441751/use-of-unaryoperator-and-binaryoperator-when-i-have-function-in-java-8